util.js

```diff
@@ -133,9 +144,10 @@ exports.isIndex = (path) => {
 /**
  * Spawn a process to run ng serve on a random open port
  */
-exports.ngServe = (publicHost, callback) => {
+exports.ngServe = (publicHost, isDist, callback) => {
```

Comment:
We don't currently use `ng serve` at all in dist mode.  Instead doing something like `ng build --prod --aot` and serving those files directly via Express.

Reply:
As we should. I'll fix this.